### ISSUE TITLE
Reranker 1: Generate query-document relevance training pairs

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#629]
# Task

Generate training data for neural reranker using query-document relevance pairs.

## Parent Epic

Part of **#616** (EPIC: Tier-Filtered Model Retraining)

## Data Source

**From existing RAG test set:**
- Ground truth question-answer pairs
- Known relevant documents from corpus

**Positive examples:**
```python
{
  "query": "Kiu fondis Esperanton?",
  "document": "Ludoviko Lazaro Zamenhof fondis Esperanton en 1887...",
  "relevance": 1.0  # Contains answer
}
```

**Negative examples:**
- Documents retrieved but don't contain answer
- Random documents from corpus

## Sampling Strategy

For each query:
- 1 positive (contains answer)
- 5 hard negatives (high lexical overlap, no answer)
- 4 random negatives

Expected: ~10K queries × 10 documents = 100K training pairs

## Output

File: `data/training/reranker_pairs.jsonl`

## Acceptance Criteria

- [ ] Positive:negative ratio = 1:9
- [ ] Hard negatives included (lexical overlap)
- [ ] Only content word roots in query/document embeddings
- [ ] File size: ~10-100MB

## Dependencies

- Blocked by: #620 (Stage 1 integration - need root embeddings)
- Blocks: #630 (Train reranker)

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
